code/game/dna/genes/gene.dm

```diff
@@ -23,30 +23,47 @@
 	// Any of a number of GENE_ flags.
 	var/flags=0
 
+	// Trait to give
+	var/trait 
+
+	// Possible activation messages
+	var/list/activation_messages=list()
```

Comment:
Space before and after `=`? Although it's inconsistent with the flags var just above it's consistent with styling most other places